src/convertToObject.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const object = {};
+  const sourceArray = sourceString.split(';').filter(el => el.length > 5);
```

Comment:
why do you filter by length?

Reply:
in order to get rid from empty elements which could be created by separating with ';'. In file stylesString.js on lines 21 and 23.

Reply:
so first trim string and then check if it's not empty. 5 is "magic" value and wouldn't work if spaces will be more

Reply:
thx